### ISSUE TITLE
Can't create table

### ISSUE THREAD

**Issue by MagnusEnger** [Kreablo/koha-plugin-vendor-acquisition#1]
Trying to install version 1.7 of the plugin on Koha 20.05.x, I get this error in /var/log/koha/koha/plack-intranet-error.log:

```
[2021/04/26 15:47:13] [WARN] DBD::mysql::db do failed: Can't create table `koha_koha`.`koha_plugin_vendoracquisition_order_json` (errno: 150 "Foreign key constraint is incorrectly formed") [for Statement "CREATE TABLE IF NOT EXISTS `koha_plugin_vendoracquisition_order_json`(
   order_id INT PRIMARY KEY NOT NULL,
   json LONGTEXT,
   FOREIGN KEY (order_id) REFERENCES `$ordertable` (order_id) ON UPDATE CASCADE ON DELETE CASCADE
) DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci
"] at /var/lib/koha/koha/plugins/Koha/Plugin/VendorAcquisition.pm line 232.
[2021/04/26 15:47:13] [WARN] Plugin Koha::Plugin::VendorAcquisition failed during installation! at /usr/share/koha/lib/Koha/Plugins/Base.pm line 58.
```

The table `koha_plugin_vendoracquisition_order_json` has not been created.

**Comment by MagnusEnger** [comment]
These tables *have* been created:

koha_plugin_vendoracquisition_default_values 
koha_plugin_vendoracquisition_item           
koha_plugin_vendoracquisition_order          
koha_plugin_vendoracquisition_record         
koha_plugin_vendoracquisition_vendormapping

**Comment by MagnusEnger** [comment]
```
FOREIGN KEY (order_id) REFERENCES `$ordertable` (order_id) ON UPDATE CASCADE ON DELETE CASCADE
```

Looks like `$ordertable` isn't getting replaced by the actual name of the table?

**Comment by AndreasJonsson** [comment]
I'm sorry, I've missed this issue report!  However, I believe this was fixed in version 1.9 of the plugin.
